Code/SimDivPickers/testPickers.cpp

```diff
@@ -0,0 +1,39 @@
+//
+//  Copyright (C) 2017 Greg Landrum
+//
+//   @@ All Rights Reserved @@
+//  This file is part of the RDKit.
+//  The contents are covered by the terms of the BSD license
+//  which is included in the file license.txt, found at the root
+//  of the RDKit source tree.
+//
+#include "MaxMinPicker.h"
+#include <iostream>
+#include <RDGeneral/Invariant.h>
+#include <RDGeneral/RDLog.h>
+#include <boost/foreach.hpp>
+
+namespace {
+  double dist_on_line(unsigned int i,unsigned int j){
+    return abs(i-j);
```

Comment:
doing abs on unsigned math is probably not doing what you want.

```
max(i, j) - min(i, j);
```

Or do an rdcast to int64